quantum/audio/audio_avr.c

```diff
@@ -110,6 +110,12 @@
 #    define TIMER_1_DUTY_CYCLE OCR1C
 #    define TIMER1_AUDIO_vect TIMER1_COMPC_vect
 #endif
+
+#if !defined(BPIN_AUDIO) && !defined(CPIN_AUDIO)
+#    error "Audio feature enabled, but no suitable pin selected - see docs/feature_audio.md under the AVR s
+ettings for available options."
```

Comment:
```suggestion
#    error "Audio feature enabled, but no suitable pin selected - see docs/feature_audio.md under the AVR settings for available options."
```

Reply:
ooops - thanks for catching that stray newline